src/OrbitQt/CaptureOptionsDialog.cpp

```diff
@@ -25,7 +25,7 @@ CaptureOptionsDialog::CaptureOptionsDialog(QWidget* parent)
   QObject::connect(ui_->buttonBox, &QDialogButtonBox::rejected, this, &QDialog::reject);
 
   if (!absl::GetFlag(FLAGS_devmode)) {
-    ui_->samplingWidget->hide();
+    ui_->samplingCheckBox->hide();
```

Comment:
Can you add a TODO comment here with the bug that is about full enable/disable of sampling.

Reply:
Done.